### DISCUSSION TITLE
alternate compilation taget ,for even faser compilation ,to lure vid-game-dev

### DISCUSSION THREAD

**Discussion by gintominto5329** [carbon-language/carbon-lang#3607] [Toolchain]
hi,
like the early days of c++ ,consider compiling to c ,and use `tcc` ,which is 6x times faster than gcc/clang ,hence the perfect target for vid-gam-dev

thanks

**Comment by chandlerc** [comment]
We already have aggressive compile time goals for the toolchain, and have good initial progress toward those goals. We think this will benefit many potential users of Carbon.
